cmsplugin_articles_ai/models/articles.py

```diff
@@ -106,6 +114,10 @@ class Article(models.Model):
         verbose_name=_("URL slug"),
         db_index=True,
     )
+    language = LanguageField(
```

Comment:
Consider the usecase:

* I want an important article to be shown on my English site (which doesn't show Finnish articles) and my Finnish site (which doesn't show English articles) without having to copy and paste the content.

How do I do that?